### PR TITLE
Update `versionName` and `versionCode` in `AndroidManifest.xml' from …

### PR DESCRIPTION
**Pull request by vmaxim**
…`package.json`

### CODE REVIEW

**Review by mykola-mokhnach — APPROVED** [review]

File: build.xml [hunk 1]
```diff
@@ -89,4 +89,10 @@
     <!-- version-tag: 1 -->
     <import file="${sdk.dir}/tools/ant/build.xml" />
 
+    <target name="set-version">
+        <replaceregexp file="AndroidManifest.xml" match='android:versionCode="(.*)"'
+            replace='android:versionCode="${env.npm_package_versionCode}"'/>
+        <replaceregexp file="AndroidManifest.xml" match='android:versionName="(.*)"'
```
Comment (mykola-mokhnach) [inline build.xml]: is it allowed to use single quotes for attribute names? I assume you still need to use double quotes and replace the stuff with `&quot;` for attribute values

Comment (vmaxim) [reply build.xml]: www.w3.org/TR/xml/#NT-AttValue

Comment (mykola-mokhnach) [reply build.xml]: does it work properly if you execute this ant command locally?

Comment (vmaxim) [reply build.xml]:
```
BUILD SUCCESSFUL
Total time: 11 seconds

C:\1\unlock_apk>%android_home%\build-tools\26.0.0\aapt dump badging bin\unlock_apk-debug.apk | findStr version*
package: name='io.appium.unlock' versionCode='3' versionName='2.2.2' platformBuildVersionName='4.4.2-1456859'
```

**Review by imurchie — APPROVED** [review]

**Pull request merged**